src/simpla-img.html

```diff
@@ -100,35 +108,36 @@
        * @return {undefined}
        */
       _styleEditable(editable) {
-        this._toggleStyles(CONDITIONAL_STYLES.editable, editable);
+        this._toggleStyles(IMG_STYLES.editable, editable);
       },
 
       /**
-       * Set inline styles for editing state
-       * @param  {Boolean} editing Value of the editing prop
+       * Set inline styles for active state
+       * @param  {Boolean} active Value of the active prop
        * @return {undefined}
        */
-      _styleEditing(editing) {
-        this._toggleStyles(CONDITIONAL_STYLES.editing, editing);
+      _styleEditing(active) {
+        this._toggleStyles(IMG_STYLES.active, active);
       },
 
       /**
        * Fake a basic placeholder with inline styles
-       * @param  {Boolean} editable Value of the editable prop
-       * @param  {Boolean} editing Value of the editable prop
-       * @param  {String} src       Value of the src prop
+       * @param  {Boolean} placeholder String to apply to background
```

Comment:
Haha, yeah I've been reading some really interesting stuff on code comments - basically like, they're a necessary evil. You should only use them if your code doesn't make sense on it's own, and you should try your hardest to make your code make sense on it's own. Regardless though, JSDoc ones are good for exporting to docs (eventually)